### ISSUE TITLE
CsvWriter must implement Closebale interface

### ISSUE THREAD

**Issue by Alexuy** [uniVocity/univocity-parsers#94]
If this wil be done, we can use AutoClosebale syntax like this:

```
try (CsvWriter writer = new CswWriter(new PrintWriter(), new CsvWriterSettings())) {
    ...
}
```

Now we must write old style code:

```
CsvWriter writer = new CswWriter(new PrintWriter(), new CsvWriterSettings());
try {
    ...
} finaly {
    writer.close();
}
```

**Comment by jbax** [comment]
Thank you for the suggestion, but the library currently supports Java 6, so this can only be implemented in a future version (3.0.0). We don't have a release date planned.

**Comment by jbax** [comment]
Duplicate of https://github.com/uniVocity/univocity-parsers/issues/71. Closing.
